packages/pipeline-editor/style/index.css

```diff
@@ -864,20 +864,26 @@ input.elyra-Dialog-checkbox.jp-mod-styled {
 
 div#root_component_parameters_cpu,
 div#root_component_parameters_memory,
+div#root_component_parameters_cpu_limit,
+div#root_component_parameters_memory_limit,
 div#root_component_parameters_gpu,
 div#root_component_parameters_gpu_vendor {
   width: 50%;
 }
 
 input#root_component_parameters_cpu,
 input#root_component_parameters_memory,
+input#root_component_parameters_cpu_limit,
+input#root_component_parameters_memory_limit,
 input#root_component_parameters_gpu,
 input#root_component_parameters_gpu_vendor {
   width: 100%;
 }
 
 #cpu,
+#cpu_limit,
 #gpu,
+#cpu_limit,
```

Comment:
@paloma-rebuelta in line 886, cpu_limit or memory_limit? I am not a CSS expert, especially also regarding the following more distinct settings for memory.